lms/templates/dashboard.html

```diff
@@ -58,6 +61,10 @@
         banner.showMessage(${json.dumps(redirect_message)})
     </%static:require_module>
   % endif
+
+    <%static:require_module module_name="js/dashboard/views/dashboard_factory" class_name="DashboardFactory">
```

Comment:
Please add this tabbed view behind a feature flag e.g. `ENABLE_TABBED_DASHBOARD`